Makefile

```diff
@@ -52,20 +54,19 @@ install-hooks:
 # Build
 
 build-models:
-	cd models \
-	&& mkdir -p dist/examples \
-	&& mkdir -p dist/schemas
-	$(foreach directory, $(wildcard models/examples/*), cp -r $(directory) models/dist/examples;)
-	$(foreach file, $(wildcard models/schemas/*.yaml), cp $(file) models/dist/schemas;)
-	$(foreach file, $(wildcard models/schemas/*.json), cp $(file) models/dist/schemas;)
-	$(foreach file, $(wildcard models/schemas/*.yaml), poetry run python scripts/yaml2json.py $(file) models/dist/schemas;)
+	$(foreach file, \
+	$(wildcard models/examples/**/SendRequest-FhirMessageSigned.json), \
+	cat $(file) \
+	| jq 'del(.entry[] | select(.resource.resourceType == "Provenance"))' \
+	| jq 'del(.entry[0].resource.focus[0])' \
+	> ""`echo $(file) | sed "s/SendRequest-FhirMessageSigned/PrepareRequest-FhirMessageUnsigned/"`"";)
```

Comment:
Stripped this back as we don't need yaml and json versions of all of these. We're using the existing symlinks to reference models directly.

Sync PrepareRequests from SendRequests, these are still validated by the fhir validator in the repo